File: source/aoBlogs2/Models/CallsToActionModel.vb

Imports Contensive.Models.Db

Namespace Models
    Public Class CallsToActionModel
        Inherits DbBaseModel
        '
        '====================================================================================================
        ''' <summary>
        '''table definition
        '''</summary>
        Public Shared ReadOnly Property tableMetadata As DbBaseTableMetadataModel = New DbBaseTableMetadataModel("Calls To Action", "callsToAction", "default", False)
        '
        '====================================================================================================
        '
        ' -- instance properties
        Public Property link As String
        Public Property headline As String
        Public Property brief As String
    End Class
End Namespace

File: source/aoBlogs2/Models/Db/CallsToActionModel.cs


using Contensive.Models.Db;

namespace Contensive.Blog.Models {
    public class CallsToActionModel : Contensive.Models.Db.DbBaseModel {
        // 
        // ====================================================================================================
        public static DbBaseTableMetadataModel tableMetadata { get; private set; } = new DbBaseTableMetadataModel("Calls To Action", "callsToAction", "default", false);
        // -- const
        // 
        // ====================================================================================================
        // -- instance properties
        public string link { get; set; }
        public string headline { get; set; }
        public string brief { get; set; }
    }
}